wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -237,13 +256,16 @@ newAccountIncludeUnready
 newAccountIncludeUnready includeUnready addGenSeed passphrase CAccountInit {..} = do
     db <- askWalletDB
     ws <- getWalletSnapshot db
+    mps <- withTxpLocalData $ \txpData -> (,)
+         <$> getLocalTxs txpData
+         <*> getLocalUndos txpData
```

Comment:
Out loud comment: if this is a general enough pattern we might want to abstract it away into a function like `getMemPoolSnapshot` (I know that's quite obvious 😀 )

Reply:
Indeed, there used to be a function called `getLocalTxsNUndos`, I believe. I think I slightly prefer the duplication here, though, to the use of functions which just pull out some subset. YMMV, though...

Reply:
I'm happy with being a bit overly explicit for the moment. It makes a nice change from the other extreme.